sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/CastSuite.scala

```diff
@@ -623,9 +624,9 @@ class CastSuite extends CastSuiteBase {
       YearMonthIntervalType()), 12)
     checkEvaluation(cast(Literal.create("INTERVAL +'1-0' YEAR TO MONTH"),
       YearMonthIntervalType()), 12)
-    checkEvaluation(cast(Literal.create(" interval +'1-0' YEAR  TO MONTH "),
+    checkEvaluation(cast(Literal.create("INTERVAL +'1-0' YEAR  TO MONTH "),
       YearMonthIntervalType()), 12)
-    checkEvaluation(cast(Literal.create(" -1-0 "), YearMonthIntervalType()), -12)
+    checkEvaluation(cast(Literal.create("-1-0"), YearMonthIntervalType()), -12)
```

Comment:
Why did you do that? This checks case sensitivity and gaps ...

Reply:
> Why did you do that? This checks case sensitivity and gaps ...

Hmmm forgot this case, change back....